### DISCUSSION TITLE
noop_visit_type should skip `TsExprWithTypeArgs`?

### DISCUSSION THREAD

**Discussion by dsherret** [swc-project/swc#8669] [General]
I was looking at this code:

https://github.com/swc-project/swc/blob/d8186fb94efb150b50d96519f0b8c5740d15b92f/crates/swc_ecma_transforms_typescript/src/strip_import_export.rs#L31-L36

It seems like that custom code could be removed if `noop_visit_type` skipped `TsExprWithTypeArgs`?

https://github.com/swc-project/swc/blob/d8186fb94efb150b50d96519f0b8c5740d15b92f/crates/swc_ecma_visit/src/lib.rs#L411

I believe `TsExprwithTypeArgs` is only used in a class' implements and an interface's extends.

I'm probably missing something though.

**Comment by kdy1** [comment]
You are right. We can add it to `noop_*_type!`. Can you send a PR?
